addons/common/XEH_postInit.sqf

```diff
@@ -326,7 +318,37 @@ enableCamShake true;
 
 // "playerInventoryChanged" event
 ["loadout", {
-    ["ace_playerInventoryChanged", [ACE_player, [ACE_player, false] call FUNC(getAllGear)]] call CBA_fnc_localEvent;
+    private _fnc_getAllGear = {
+        if (isNull _this) exitWith {[
+            "",
+            "",
+            "", [],
+            "", [],
+            "", [],
+            "", ["","","",""], [],
+            "", ["","","",""], [],
+            "", ["","","",""], [],
+            [],
+            "",
+            ""
+        ]};
+
+        [
+            headgear _this,
+            goggles _this,
+            uniform _this, uniformItems _this,
+            vest _this, vestItems _this,
+            backpack _this, backpackItems _this,
+            primaryWeapon _this, primaryWeaponItems _this, primaryWeaponMagazine _this,
+            secondaryWeapon _this, secondaryWeaponItems _this, secondaryWeaponMagazine _this,
+            handgunWeapon _this, handgunItems _this, handgunMagazine _this,
+            assignedItems _this,
+            binocular _this,
+            _this call CBA_fnc_binocularMagazine
+        ]
+    };
+
+    ["ace_playerInventoryChanged", [ACE_player, ACE_player call _fnc_getAllGear]] call CBA_fnc_localEvent;
```

Comment:
`CBA_fnc_localEvent` != `CBA_fnc_addEventHandler`

I want to pass the return value of `_fnc_getAllGear` and not some \<CODE\>